infra/modules/service/main.tf

```diff
@@ -286,6 +291,15 @@ resource "aws_cloudwatch_log_group" "service_logs" {
   # TODO(https://github.com/navapbc/template-infra/issues/164) Encrypt with customer managed KMS key
   # checkov:skip=CKV_AWS_158:Encrypt service logs with customer key in future work
 }
+####################
+## Logging Bucket ##
+####################
+
+module "alb_logging" {
+  source           = "../s3-encrypted"
+  environment_name = var.service_name
```

Comment:
Setting the `environment_name` to the service_name seems wrong. We don't pass the environment name to this module and I don't think I want to. I think we're only using the s3-encrypted module's `environment_name` variable for the logging [target_prefix](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_logging#target_prefix):

>  - (Required) Prefix for all log object keys.

So it's probably better to rename this variable to be something like "logging_bucket_target_prefix"

Reply:
we've also hardcoded into the s3-encrypted module:

```
  description = "KMS key for ${var.environment_name} Document Upload"
```

And I think we should remove the "Doc upload" part since we're using this module for more than just doc upload

Reply:
I'm actually making some changes to this module, so I'll incorporate these into my PR and tag you on it.